shell/platform/darwin/ios/framework/Source/FlutterDartProject.mm

```diff
@@ -337,4 +337,8 @@ + (NSString*)defaultBundleIdentifier {
   return @"io.flutter.flutter.app";
 }
 
+- (void)setLeakDartVM:(BOOL)enabled {
+  _settings.leak_vm = enabled;
```

Comment:
Is it better to read `leak_vm` from the main bundle's meta-plist? like https://github.com/eggfly/engine/blob/disable_leak_vm/shell/platform/darwin/ios/framework/Source/FlutterDartProject.mm#L160-L161
```
  // SkParagraph text layout library
  NSNumber* enableSkParagraph = [mainBundle objectForInfoDictionaryKey:@"FLTEnableSkParagraph"];
  settings.enable_skparagraph = (enableSkParagraph != nil) ? enableSkParagraph.boolValue : false;
```

Keep the same way with Android. Not add a public setter method.

Reply:
Done~